enterprise/server/remote_execution/workspace/workspace_unix.go

```diff
@@ -0,0 +1,11 @@
+//go:build !windows
```

Comment:
If you use `//go:build (linux || darwin) && !android && !ios` it will reduce the noise in the BUILD file.